lib/kafo/configuration.rb

```diff
@@ -30,7 +30,9 @@ class Configuration
         :facts                => {},
         :low_priority_modules => [],
         :verbose_log_level    => 'info',
-        :skip_puppet_version_check => false
+        :skip_puppet_version_check => false,
+        :parser_cache_path    => nil,
+        :ignore_undocumented  => nil,
```

Comment:
Just comparing to https://github.com/theforeman/foreman-installer/blob/develop/config/foreman.yaml I think you're missing:
* `hiera_config`
* `log_owner`
* `log_group`
* `config_header_file`
* `dont_save_answers`
* `kafo_modules_dir`
* `order`

There may be more.